miio/miot_device.py

```diff
@@ -30,7 +30,7 @@ class MiotDevice(Device):
 
     mapping = None
 
-    def get_properties_for_mapping(self) -> list:
+    def get_properties_for_mapping(self, max_properties=15) -> list:
```

Comment:
```suggestion
    def get_properties_for_mapping(self, *, max_properties=15) -> list:
```
let's make this kwarg only to make the code more readable (even when the name is also a bit off..)?